src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -370,7 +374,7 @@ synchronized public void createPythonInterpreter() {
     }
     log.info("Python System Path: {}", sys.path);
 
-    String selfReferenceScript = "from org.myrobotlab.service import Runtime\n" + "from org.myrobotlab.service import Python\n"
+    String selfReferenceScript = "from org.myrobotlab.framework import Platform\n" + "from org.myrobotlab.service import Runtime\n" + "from org.myrobotlab.service import Python\n"

```

Comment:
how about just do   "from org.myrobotlab.service import *"    ?

Reply:
I likey ! :)
Ahahahah ... crap .. Ya great idea - but it doesn't work ... raise a Jython issue perhaps ;)